### PR TITLE
Handle emails from mailgun [OSF-6737]

### PR DESCRIPTION
**Pull request by jnayak1**
-Create an endpoint that mailgun can post to.
-Email user with confirmation that the submission was successful. 
-Handle errors, sending an email to the user saying that the email was unsuccessful with explanation for why it was unsuccessful.

Notes:
-Will need mailgun api key to actually send the email

### CODE REVIEW

File: meetings/mail/models.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# from __future__ import unicode_literals
+
+# from django.db import models
```
Comment (Ghalko) [inline meetings/mail/models.py]: Remove this file for now.

File: meetings/mail/admin.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# from django.contrib import admin
```
Comment (Ghalko) [inline meetings/mail/admin.py]: Remove this file for now.

File: meetings/mail/templates/conference_does_not_exist.mako [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Hello {{fullname}},
```
Comment (Ghalko) [inline meetings/mail/templates/conference_does_not_exist.mako]: Convert mako files to just html files. No need to scare people with yet another potential renderer.

File: meetings/mail/urls.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+from django.conf.urls import include, url
+
+urlpatterns = [
+    url(r'^', include('django_inbound_email.urls')),
```
Comment (Ghalko) [inline meetings/mail/urls.py:4]: Is this from a package?

File: meetings/meetings/settings/local-dist.py [hunk 5]
```diff
@@ -5,6 +5,10 @@
 DEBUG = True
 CLIENT_ID = ''
 CLIENT_SECRET = ''
+MAILGUN_API_KEY = ''
+ANYMAIL = {
+    "MAILGUN_API_KEY": MAILGUN_API_KEY,
+}
```
Comment (Ghalko) [inline meetings/meetings/settings/local-dist.py]: Put these in defaults.py and comment that they need to be completed in local.py.

**Comment by Ghalko** [comment]
@jnayak1 Pass done.

File: meetings/mail/urls.py (hunk 4 above)
Comment (jnayak1) [inline meetings/mail/urls.py:4]: yes,
https://pypi.python.org/pypi/django-inbound-email/

Comment (Ghalko) [inline meetings/mail/urls.py:4]: OK good.

File: meetings/mail/templates/conference_does_not_exist.mako (hunk 3 above)
Comment (jnayak1) [inline meetings/mail/templates/conference_does_not_exist.mako]: How do we pass the links to nodes or conferences without a renderer? @Ghalko

Comment (Ghalko) [inline meetings/mail/templates/conference_does_not_exist.mako]: mako is a specific renderer. Django has its own built in renderer already.

**Comment by jnayak1** [comment]
Using the django-inbound-email package, you can't get the name of the person sending the email.

**Pull request closed without merging**